tests/tck/features/match/Base.IntVid.feature

```diff
@@ -575,3 +575,17 @@ Feature: Basic match
       RETURN count(names)
       """
     Then a SemanticError should be raised at runtime: To get the property of the vertex in `v3.name', should use the format `var.tag.prop'
+
+  Scenario: filter is not a valid expression
+    When executing query:
+      """
+      MATCH (v:player) WHERE v.player.name-'n'=="Tim Duncann" RETURN v
+      """
+    Then a SemanticError should be raised at runtime: `(v.player.name-"n")' is not a valid expression, can not apply `-' to `__EMPTY__' and `STRING'.
```

Comment:
This error message is confusing. Where is the `__EMPTY__`?

Reply:
The error message will try to evaluate the expression and print its type in the message. This is to unify the error messages we currently using.